docs/features/dataset-usage-and-query-history.md

```diff
@@ -0,0 +1,75 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Dataset Usage & Query History
+
+<FeatureAvailability/>
+
+Dataset Usage & Query History can give dataset-level information about the top queries which referenced a dataset.
+
+Usage data can help identify the top users who probably know the most about the dataset and top queries referencing this dataset.
+You can also get an overview of the overall number of queries and distinct users.
+In some sources, column level usage is also calculated, which can help identify frequently used columns.
+
+With sources that support usage statistics, you can collect Dataset, Dashboard, and Chart usages.
+
+## Dataset Usage & Query History Setup, Prerequisites, and Permissions
+
+To ingest Dataset Usage & Query History data, you should check first on the specific source doc
+if it is supported by the Datahub source and how to enable it.
+
+You can validate this on the Datahub source's capabilities section:
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/source-snowflake-capabilities.png"/>
+</p>
+
+There are some sources where you have to use a different usage specific source for usage ingestion. In this
+case it is noted on the capabilities summary like in the example below.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/source-redshift-capabilities.png"/>
+</p>
+
+Please, always check the usage prerequisities page if the source has as it can happen you have to add additional
+permissions which only needs for usage.
+
+## Using Dataset Usage & Query History
+
+After successful ingestion, the Query tab will be enabled on datasets with any usage.
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/feature-queries-tab.png"/>
+</p>
+
+On the query tab, you can see the top 5 queries which referenced this dataset.
```

Comment:
edit: On the Queries tab

Reply:
Suggestion: can we add context about how we choose the top 5 queries? is it based on number of times executed? over what period of time?